map/map_loader/src/pointcloud_map_loader/pointcloud_map_loader_node.cpp

```diff
@@ -33,22 +33,24 @@
 #include <glob.h>
 #include <pcl/io/pcd_io.h>
 #include <pcl_conversions/pcl_conversions.h>
-#include <rcutils/filesystem.h>  // To be replaced by std::filesystem in C++17
 
+#include <filesystem>
 #include <string>
 #include <vector>
 
+namespace fs = std::filesystem;
+
 namespace
 {
 bool isPcdFile(const std::string & p)
 {
-  if (!rcutils_is_file(p.c_str())) {
+  if (fs::is_directory(p)) {
     return false;
   }
 
-  const auto ext = p.substr(p.find_last_of(".") + 1);
+  const std::string ext = fs::path(p.c_str()).extension();
```

Comment:
Do you need `c_str()` here?

```suggestion
  const std::string ext = fs::path(p).extension();
```

Reply:
Not necessary. Sorry, I'll fix it.